README.md

```diff
@@ -22,6 +22,86 @@ Add the following code to the `pom.xml` file of the project where you want to ex
 		<version>1.0-SNAPSHOT</version>
 		<configuration>
 			<agentPath>{path_to_opentelemetry-javaagent-all.jar}</agentPath>
+
+			<exporter>
+				<otlpExporter>
```

Comment:
I noticed they added some parameters to this exporter in recent commits (e.g. protocol and certificate). We should probably add these as well.